### PR TITLE
fix(file): ignore errors due to missing file

### PR DESCRIPTION
**Pull request by open-dynaMIX**
If no file is uploaded to the storage service, we want to ignore the resulting
exceptions in order to still be able to modify/delete the File.

### CODE REVIEW

File: caluma/caluma_form/tests/test_model.py [hunk 1]
```diff
@@ -34,3 +34,29 @@ def test_update_file(db, file_factory, mocker):
     file.save()
     Minio.copy_object.assert_called()
     Minio.remove_object.assert_called()
+
+
+@pytest.mark.parametrize("should_raise", [False, True])
+def test_missing_file(db, file, mocker, should_raise):
+    exc_code = "NoSuchKey"
+    if should_raise:
+        exc_code = "SomeOtherError"
```
Comment (sbor23) [inline caluma/caluma_form/tests/test_model.py:43]: You don't like python's ternary? :)

**Review by sbor23 — COMMENTED** [review]
Just a curious comment.

What's the reasoning behind keeping the file and renaming the pk in `_move_blob`?
We want to keep all historical versions of a file?

File: caluma/caluma_form/tests/test_model.py (hunk 1 above)
Comment (open-dynaMIX) [reply caluma/caluma_form/tests/test_model.py]: Changed, thx

**Comment by open-dynaMIX** [comment]
> We want to keep all historical versions of a file?

Exactly

**Review by sbor23 — APPROVED** [review]

**Pull request merged**